examples/complete/main.tf

```diff
@@ -5,7 +5,7 @@ provider "aws" {
 module "codebuild" {
   source                      = "../../"
   description                 = "This is my awesome CodeBuild project"
-  concurrent_build_limit      = 10
+  concurrent_build_limit      = 1
```

Comment:
Any reason you needed to make this change?

Reply:
Yes, having a value different than 1 made the test fail in my testing AWS account:

> Error creating CodeBuild project: InvalidInputException: Project-level concurrent build limit cannot exceed the account-level concurrent build limit of 1.

As 1 seems to be the [default quota](https://docs.aws.amazon.com/codebuild/latest/userguide/limits.html) for new AWS accounts, I would expect this change to reduce failures without any downsides (unless we want to start testing that the actual Codebuild project was created with the specified concurrency).